File: ruby/src/azure/mgmt/microsoft_compute/2016_06_30/operations/disks_operations.rb
require_relative "../models/disk"

module Azure
  module Mgmt
    module Compute
      module Api_2016_06_30
        module Operations
          class DisksOperations
            def initialize(client)
            end

            def create(name:, location:, tags: nil, disk_size_gb: nil, owner_id: nil)
              Azure::Mgmt::Compute::Api_2016_06_30::Models::Disk.new(name: name,
                                                                     location: location,
                                                                     tags: tags,
                                                                     disk_size_gb: disk_size_gb,
                                                                     owner_id: owner_id)
            end
          end
        end
      end
    end
  end
end

File: dotnet/src/Azure/Mgmt/Microsoft.Compute/2016_06_30/Operations/DisksOperations.cs
using Azure.Mgmt.Models;
using Azure.Mgmt.Compute._2016_06_30.Models;
using System.Collections.Generic;

namespace Azure.Mgmt.Compute._2016_06_30.Operations {
    public class DisksOperations {
        internal DisksOperations(AzureClient client){

        }

        public IDisk Create(string name, string location, IDictionary<string, string> tags = null, int? diskSizeGB = null, string ownerId = null){
            return new Disk(name, location, tags, diskSizeGB, ownerId);
        }
    }
}